package.json

```diff
@@ -77,10 +78,10 @@
   "devDependencies": {
     "@babel/core": "^7.6.2",
     "@babel/preset-env": "^7.6.2",
-    "@bigcommerce/tslint-config": "^4.0.0",
+    "@bigcommerce/eslint-config": "^1.0.1",
```

Comment:
Are you going to do the upgrade in a separate PR?

Reply:
Will do upgrade in a seperate PR since there's about 3000+ errors to fix up